.github/workflows/generate_decendant_mapping.yml

```diff
@@ -0,0 +1,57 @@
+#This is a github actions that run every monday at 12am. It will run generate_decendant_mapping.py off the main
+#  github branch and open a pull request to commit the new ontology_decendant files to cellxgene-ontology-guide
+
+name: Generate Ontology Decendant Mappings
+on:
+  #  schedule:
+  #    - cron: "0 0 * * 1"
+  pull_request:
+    branches:
+      - tsmith/decendent-mappings # TODO remove this line after testing
```

Comment:
remember to update this!

Reply:
Also--let's add a workflow_dispatch option to have the option to run it manually (in case we want to do it off-schedule or just for testing the gha)